snuba/util.py

```diff
@@ -144,6 +148,8 @@ def column_expr(dataset, column_name, body, alias=None, aggregate=None):
 
     if is_function(column_name, 0):
         return complex_column_expr(dataset, column_name, body)
+    elif isinstance(column_name, (list, tuple)) and aggregate:
+        return complex_column_expr(dataset, [aggregate, column_name, alias], body)
```

Comment:
does it work if you just call this directly from api.py skipping column_expr ? Ideally this would work like for line 149 for functions, but for some reason we are already unpacking the expression before calling column_expr for aggregates so we cannot do that.

Reply:
Going directly to complex_column_expr from api.py ends up breaking aggregation handling for things like `['topK(4)', 'issue', 'aggregate']` as the `topK(4)` pattern isn't recognized as a function.